File: algorithms/1134/main.go
func isArmstrong(N int) bool {
    s := fmt.Sprintf("%v", N)
    l := float64(len(s))
    sum := 0
    for _, c := range s {
        p, _ := strconv.Atoi(string(c))
        sum += int(math.Pow(float64(p), l))
    }
    return N == sum
}

File: algorithms/1134/main.scala
import scala.math.pow

object Solution {
    def isArmstrong(N: Int): Boolean = {
        (for (n <- N.toString) yield {println(n.toString.toInt, N.toString.length);pow(n.toString.toInt, N.toString.length)}).sum.toInt == N
    }
}
